1001-2000/LIT1509GadlaW.xml

```diff
@@ -47,6 +44,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       </encodingDesc>
       <profileDesc>
          <langUsage><language ident="en">English</language><language ident="gez">Gǝʿǝz</language></langUsage>
+         <abstract><p>
+            Although no manuscript containing only the vita (gadl) has been reported thus far, this record pertains solely to the vita. It is distinguished from the miracles, as the set of miracles following the vita can vary. Furthermore, the gadl and miracles are visually and logically distinct.
+         </p></abstract>
```

Comment:
```
The Life of `<persName ref="PRS9848Walatta"/>`, usually followed by a set of <ref type="work" corresp="LIT7073TammeraW">Miracles</ref>. 
Although no manuscript containing only the vita (gadl) has been reported thus far, it is distinguished from the <ref type="work" corresp="LIT7073TammeraW">miracles</ref>, as the set of miracles following the vita can vary. Furthermore, the gadl and miracles are visually and logically distinct.
```

Reply:
good, I suppose

Reply:
@karljonaskarlsson, @eu-genia, from my perspective, I think I have addressed your suggestions and remarks. May I ask you to check it once again, and if all is fine, allow the merge?